utils/eventtracking.js

```diff
@@ -117,8 +127,8 @@ const askForConsentIfNeeded = async (options) => {
 
 const trackDeployedContract = async () => {
     const shellSettings = settings.getShellSettings();
-    const id = getMixpanelID(shellSettings);
-    await mixpanel.people.increment(id, 'deployed_contracts');
+    const id = getSegmentID(shellSettings);
+    // TODO: find a way to increment a Segment analytic property
```

Comment:
@itegulov are you planning to work on it now?
Deployed contracts are the most important part of analytics IMO.

Reply:
I have spent some time looking for alternatives to no avail, so I think it would make sense to hear @TiffanyGYJ's opinion first before spending even more time on this

Reply:
> Deployed contracts are the most important part of analytics IMO.

Curious to hear your reasoning. Wouldn't aggregating `deployed` events achieve the same goal? Do we need to store/cache this as a definite number for something?

Reply:
Deployed events might happen multiple times with different contracts, so that might not be an accurate estimate on how many unique contracts each dev deployed. 
Is there a way we can track that somewhere else or finding a way from Segment/Mixpanel is the best option?

Reply:
@TiffanyGYJ 
> Deployed events might happen multiple times with different contracts, so that might not be an accurate estimate on how many unique contracts each dev deployed.

Looking through the code, the current logic does not track unique deployed contracts either, just the number of times when `devDeploy`/`deploy` were executed successfully.

Reply:
I think that the total number of deploys per week is the most important metric since we are tracking developer activity.
The number of active contracts (contracts that have been deployed or redeployed in the past week) is also important.
Tracking a particular user and his deploys is not that important IMO, and a bit shady.

Reply:
> I think that the total number of deploys per week is the most important metric since we are tracking developer activity. The number of active contracts (contracts that have been deployed or redeployed in the past week) is also important.

I agree with this, but a per-user counter does not really help with that, no? Increments do not have a timestamp, so you would have to track how much a counter has changed each week which kind of beats its point. I think we can deduce all that information just from tracking events (by writing a small custom event consumer or building something in Segment itself, not sure if it provides any processing logic).

> Tracking a particular user and his deploys is not that important IMO, and a bit shady.

:+1:

Reply:
Agree!